cloud/wfm-rest-api/example/index.ts

```diff
@@ -0,0 +1,33 @@
+import { getLogger } from '@raincatcher/logger';
+import * as express from 'express';
+import { MongoClient } from 'mongodb';
+import * as path from 'path';
+import { WfmRestApi } from '../src/index';
+
+const app = express();
+
+// Create api
+const api = new WfmRestApi();
+
+// Mount api into path
+app.use('/api', api.createWFMRouter());
+
+// Use connect method to connect to the server
+const url = 'mongodb://localhost:27017/raincatcher';
+MongoClient.connect(url, function(err, db) {
+  if (db) {
+    api.setDb(db);
+  } else {
+    console.info('MongoDb not connected', err);
+    process.exit();
```

Comment:
Should add some non-zero value to make sure the process exits with an error: `process.exit(1)`.

Alternatively this could be done before starting the http server at all:
`promisify(MongoClient.connect)(...).then(db => api.setDb(db)).then(() => app.listen(...));`